templates/marketplace/dsm-mp.template

```diff
@@ -251,6 +251,8 @@ Mappings:
     sa-east-1:
       PerHost: ami-1f275373
       BYOL: ami-3d057151
+    us-gov-west-1:
+      BYOL: ami-f5de5e94
```

Comment:
Is this prevented via a rule/condition and/or documented in the deployment guide?